spec/requests/simple_smart_answers_spec.rb

```diff
@@ -15,6 +15,8 @@ def simple_smart_answer_content_item
       content_store_has_random_item(base_path: "/the-bridge-of-death", schema: "simple_smart_answer")
     end
 
+    it_behaves_like "it can render the govuk_chat promo banner", "/the-bridge-of-death"
```

Comment:
Minor: The chat promo banner has been removed, so this example no longer exists.

Reply:
Fixed